### PR TITLE
Improve generic makefile

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: common/makefiles/generic-make-go.mk [hunk 1]
```diff
@@ -32,7 +32,7 @@ ARTIFACTS:=/tmp/artifacts
 endif
 
 # Base docker configuration
-DOCKER_CREATE_OPTS := -v $(LOCAL_DIR):$(WORKSPACE_LOCAL_DIR):delegated -v $(ARTIFACTS):/tmp/artifacts --rm -w $(WORKSPACE_COMPONENT_DIR) $(BUILDPACK)
```
Comment (dbadura) [inline common/makefiles/generic-make-go.mk:35]: revert this

**Comment by dbadura** [comment]
pre-kyma-components-application-gateway

File: common/makefiles/generic-make-go.mk [hunk 2]
```diff
@@ -78,7 +78,11 @@ endef
 verify:: test check-imports check-fmt
 format:: imports fmt
 
-release: resolve dep-status verify build-image push-image
+release:
+	$(MAKE) release-old
```
Comment (dbadura) [inline common/makefiles/generic-make-go.mk:82]: release-dep

**Review by Tomasz-Smelcerz-SAP — APPROVED** [review]

**Review by piotrmiskiewicz — APPROVED** [review]

**Review by piotrmiskiewicz — APPROVED** [review]

**Pull request merged**